libunwind/src/UnwindLevel1.c

```diff
@@ -170,6 +183,10 @@ unwind_phase1(unw_context_t *uc, unw_cursor_t *cursor, _Unwind_Exception *except
 }
 extern int __unw_step_stage2(unw_cursor_t *);
 
+#if defined(_LIBUNWIND_USE_CET) && defined(_LIBUNWIND_TARGET_AARCH64)
```

Comment:
Mixing CET and AArch64 could cause confusion. Could AArch64 use a different variable?